packages/scaffold-ui/src/views/w3m-connecting-farcaster-view/index.ts

```diff
@@ -93,6 +101,37 @@ export class W3mConnectingFarcasterView extends LitElement {
     </wui-flex>`
   }
 
+  private loadingTemplate() {
+    return html`<wui-flex
+    flexDirection="column"
```

Comment:
Is this file formatted? Looks interesting that props does have the same space with `return` 🤔